.eslintignore

```diff
@@ -3,5 +3,6 @@ src/vendor/**/*.js
 src/test/utils.js
 *.test.js
 example/
+sandbox/
```

Comment:
Actually, I think it's worth it to run prettier and eslint on the sandbox JS files

Reply:
@philpl do I also need to add flow annotation to the sandbox, or this eslint rule could be ignored?

Reply:
@gribnoysup oh good point! no, I don't think we'll need Flow on the sandbox. Maybe it's worth then to separate the prettier/eslint setup a little